src/coreclr/jit/objectalloc.cpp

```diff
@@ -3204,6 +3204,9 @@ GenTree* ObjectAllocator::IsGuard(BasicBlock* block, GuardInfo* info)
     bool isNonNull = false;
```

Comment:
nit for style:
```diff
- GenTree* ObjectAllocator::IsGuard(BasicBlock* block, GuardInfo* info)
+ /* static */ GenTree* ObjectAllocator::IsGuard(BasicBlock* block, GuardInfo* info)
```

Reply:
Thanks. I will add this to my (growing) list of deferred cleanups.